src/idl/events.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
```

Comment:
This file, `events.proto`, is copied verbatim from the main repo. You should disregard it.